### DISCUSSION TITLE
Rec for formatting/styling tables jupyter with docx output?

### DISCUSSION THREAD

**Discussion by adpadi** [quarto-dev/quarto-cli#1110] [Q&A] [answered]
My pandas styler formatted tables from jupyter don't have any formatting when outputting to word docx. Is there a recommendation for how to format tables for docx output?

**Comment by jjallaire** [comment, marked as answer]
There isn't yet a python tables package we know of that does rich formatting for docx (we are hoping to soon work on a python version of the R gt package which will do some of this)
